### PR TITLE
Update project-maintainers.csv for containerd

### PR DESCRIPTION
**Pull request by samuelkarp**
This brings project-maintainers.csv in line with https://github.com/containerd/project/blob/main/MAINTAINERS, which is where the containerd project keeps track of maintainers.

### CODE REVIEW

**Comment by dims** [comment]
LGTM

File: project-maintainers.csv [hunk 1]
```diff
@@ -69,17 +69,34 @@ Graduated,CoreDNS,Chris O'Haver,Infoblox,chrisohaver,https://github.com/coredns/
...
+,,Xuean Yan,ZTE,yanxuean,
+,,Yanqiang Miao,ZTE,miaoyq,
+,,Evan Hazlett,Kentik Labs,ehazlett,
+,,Ace Tang,Alibaba,Ace-Tang,
+,,Brian Goff,Microsoft,cpuguy83,
+,,Sebastiaan van Stijn,Docker,thajeztah,
+,,Boris Popovschi,ArangoDB,Zyqsempai,
+,,Kohei Tokunaga,NTT,ktock,
+,,Daniel Canter,Microsoft,dcantah,
+,,Michael Zappa,Stateless,MikeZappa87,
```
Comment (estesp) [inline project-maintainers.csv:99]: Michael is now at Microsoft

**Comment by estesp** [comment]
one nit about a recent change in employer for one of our new reviewers; otherwise LGTM

**Comment by AkihiroSuda** [comment]
> Note that the MAINTAINERS file includes both committers and reviewers,

Other projects seems only listing people with the merge access?

**Comment by samuelkarp** [comment]
Updated per discussion in the maintainers' slack channel.  This now only includes committers.

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**